cypress/integration/brewery/ScenarioValidation.spec.js

```diff
@@ -4,38 +4,43 @@
 import utils from '../../commons/TestUtils';
 import { DATASET, RUN_TEMPLATE } from '../../commons/constants/brewery/TestConstants';
 import { Scenarios, ScenarioManager, ScenarioParameters, Login } from '../../commons/actions';
-
-Cypress.Keyboard.defaults({
-  keystrokeDelay: 0,
-});
+import { ALL_ROOT_SCENARIOS } from '../../fixtures/stubbing/ScenarioListOrder/scenarios';
```

Comment:
Please don't use static data from other tests, because it will be harder to maintain tests when we have to fix specific fixtures (i.e. `fixtures/stubbing/ScenarioListOrder/scenarios` should only be used in the _ScenarioListOrder_ test).

By default, the stubbing class is initialized with a default list of scenarios, so you don't need to import and use `ALL_ROOT_SCENARIOS`